scripts/src/main/resources/scripts/command/ide

```diff
@@ -377,14 +377,14 @@ function doUpdateSoftware() {
   for tool in "${DEVON_IDE_TOOLS[@]}"
   do
     tool_path="${DEVON_IDE_HOME}/software/${tool}"
-    if [ "${tool}" == "maven" ]
+    if [ "${tool}" == "mvn" ]
```

Comment:
OMG, this was indeed swapped.
You found a stupid bug here and fixed it. Thanks!